### ISSUE TITLE
Thanks!

### ISSUE THREAD

**Issue by coderschoolreview** [liiintz/Homework---Linh#1]
## The goal of this assignment is focus on:

- Take a step further into visualization world.
- How to split your data into train and test part
- Correlation between each feature columns and label column
- Build your first model linear regression and try some methods to evaluate (MAE, MSE, RMSE)
- Make some insights and conclusion about the data

## Things you did well:

- You can actually load data and perform simple visualization steps.
- More familiar with seaborn, it's a library built on top matplotlib, very powerful and great potential for visualize task. Visualization plays an important role in data science. It's not only help you understand data but also a good way to explain the problem to your client, your boss.

## Things to work on:
- You need to practice more and complete the whole assignment as it will provide you useful experience. :) 
- Try your best to get used to the concept of training+testing set and fitting machine learning model.

Keep up the good work. You’re getting better and better.

**Comment by coderschoolreview** [comment]
### The goal of this assignment is focus on
- Get to know about NLP techniques
- Understand the concept of classification problem
- Evaluate the classification result with Accuracy, Precision and Recall
- Build your first model logistic regression

### Things you did well:
- Understand the basic flow to deal with classification task
- Be familiar with the techniques of NLP
- You get used to dealing with train/test split concept and evaluating results with accuracy reports

### Things to work on:
- You need to understand the meaning of each step in this exercise (not only copy and paste)
- Revise the notebook to be more comprehensive and extract more information from the results
- You should recap everything you understand, not only just run codes.

In summary, you did a great job and improved very significantly :). Keep up with the final project !!!.
